packages/bridge-ui/src/utils/recommendProcessingFee.spec.ts

```diff
@@ -0,0 +1,192 @@
+const mockChainIdToTokenVaultAddress = jest.fn();
+jest.mock("../store/bridge", () => ({
```

Comment:
should this be below the imports? stylistically

Reply:
no, this mock of the partial page actually has to be above imports